packages/react/src/schematics/init/init.ts

```diff
@@ -1,9 +1,11 @@
 import { chain, noop, Rule } from '@angular-devkit/schematics';
+import { JsonObject } from '@angular-devkit/core';
```

Comment:
This is just moved from below, an old habit of grouping external imports before internal ones